apps/web/components/tailwind/extensions.ts

```diff
@@ -106,6 +112,16 @@ const starterKit = StarterKit.configure({
   gapcursor: false,
 });
 
+const lowlight = createLowlight(common);
+lowlight.register("html", html);
+lowlight.register("css", css);
+lowlight.register("js", js);
+lowlight.register("ts", ts);
+
+const codeBlockLowlight = CodeBlockLowlight.configure({
+  lowlight,
+});
+
```

Comment:
Nice, I've been looking at implementing this feature!
I think this would need a way to specify languages via configuration, since this block basically just hardcodes these 4

Reply:
Related: https://github.com/steven-tey/novel/issues/369

Reply:
I have updated the PR.

Simplified the language support and removed the hardcoded ones. 

Refer: https://github.com/wooorm/lowlight?tab=readme-ov-file#common

`createLowlight(common)` would cover 37 languages by default, which should be good enough for most of the use cases.

Reply:
I have updated the PR.

Simplified the language support and removed the hardcoded ones. 

Refer: https://github.com/wooorm/lowlight?tab=readme-ov-file#common

`createLowlight(common)` would cover 37 languages by default, which should be good enough for most of the use cases.